helper/geojsonify.js

```diff
@@ -179,13 +179,13 @@ function copyProperties( source, props, dst ) {
 }
 
 /**
- * Determine and set place id, type, and source
+ * Determine and set place gid, type, and source
  *
  * @param {object} src
  * @param {object} dst
  */
 function addMetaData(src, dst) {
-  dst.id = src._id;
+  dst.gid = src._type + ':' + src._id;
```

Comment:
It's not the most complicated line of code, but it might be nice to move this to a createGid helper method. Thoughts?